Formula/dcmtk.rb

```diff
@@ -17,20 +17,23 @@ class Dcmtk < Formula
 
   option "with-docs", "Install development libraries/headers and HTML docs"
   option "with-libiconv", "Build with brewed libiconv. Dcmtk and system libiconv can have problems with utf8."
+  option "with-dicomdict", "Build with baked-in DICOM data dictionary."
+  option "without-openssl", "Build without support for OpenSSL."
```

Comment:
The default option text is good enough not to need this line really. Just use:

```ruby
depends_on "openssl" => :recommended
```
I'm pretty sure it would never be merely `build` either way unless it is doing a static build.